src-cpp/KafkaConsumerImpl.cpp

```diff
@@ -172,6 +183,24 @@ RdKafka::KafkaConsumerImpl::assign (const std::vector<TopicPartition*> &partitio
 
 
 RdKafka::ErrorCode
+RdKafka::KafkaConsumerImpl::redirectPartitionQueue (const TopicPartition *partition,
```

Comment:
Lets use the same name as in the C code as to make documentation references agnostic. (e.g. setPartitionQueue)